agentops/llm_tracker.py

```diff
@@ -10,6 +11,7 @@
 
 class LlmTracker:
     SUPPORTED_APIS = {
+        'litellm': {'1.0.0': ("openai_chat_completions.completion",)},  # TODO
```

Comment:
Have you tested v1.0.0? [Optional] Might be good to just confirm that it works back till 1.0.0.

Reply:
Good call. It actually doesn't work until 1.3.1
1.0.0 - 1.2.0 send the response as __dict__ instead of Pydantic so we get this error
<img width="1013" alt="Screenshot 2024-04-25 at 6 36 22 PM" src="https://github.com/AgentOps-AI/agentops/assets/15098079/b3ea3380-8f7e-4674-9788-5bb61621eaeb">

Though in finding this out I found that there will be misalignments between us and LiteLLM